.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,27 @@
-- [ ] Is it mergeable?
-- [ ] `make test` Did it pass the tests?
-- [ ] `make clean diff-cover` If it introduces new functionality in
-  `scripts/` is it tested?
```

Comment:
No need to burden the users with these first 3 points. The GitHub interface won't allow unmergeable merges, Travis CI runs the test build, and CodeCov gives a coverage report.